docs/recipes/low_freq_proc.qmd

```diff
@@ -0,0 +1,183 @@
+
+# Apply the low-pass filter on the delta patch
+delta_pa_low_passed = delta_pa.pass_filter(time=(None, cutoff_freq * filter_safety_factor))
+# Resample the low-passed filter patch
+new_time_ax = np.arange(delta_pa.attrs["time_min"], delta_pa.attrs["time_max"], np.timedelta64(dt, "s"))
+delta_pa_lfp = delta_pa_low_passed.interpolate(time=new_time_ax)
+
+# Identify the indices where the absolute value of the data exceeds the threshold
+data_abs = np.abs(delta_pa_lfp.data)
+threshold = np.max(data_abs) * tolerance
+ind = data_abs > threshold
+ind_1 = np.where(ind)[1][0]
+ind_2 = np.where(ind)[1][-1]
+
+# Get the total duration of the processed delta function patch in seconds
+delta_pa_lfp_seconds = (delta_pa_lfp.attrs["time_max"] - delta_pa_lfp.attrs["time_min"]) / np.timedelta64(1, "s")
+# Convert the new time axis to absolute seconds, relative to the first timestamp
+time_ax_abs = (new_time_ax - new_time_ax[0]) / np.timedelta64(1, "s")
+# Center the time axis 
+time_ax_centered = time_ax_abs - delta_pa_lfp_seconds // 2 
+
+# Calculate the maximum of edges in both sides (in seconds) where artifacts are present
+edge = max(np.abs(time_ax_centered[ind_1]), np.abs(time_ax_centered[ind_2]))
+
+# Validate the `edge` value to ensure sufficient processing patch size
+if np.ceil(edge) >= chunk_size / 2:
+    raise ValueError(
+        f"The calculated `edge` value ({edge:.2f} seconds) is greater than half of the processing patch size "
+        f"({chunk_size:.2f} seconds). To resolve this and increase efficiency, consider one of the following:\n"
+        "- Increase `memory_size` to allow for a larger processing window.\n"
+        "- Increase `tolerance` to reduce the sensitivity of artifact detection."
+    )
+```
+
+
+## Perform low-frequency processing and save results on disk
+```{python}
+# Helper functions to handle the name of LF processed patches 
+def _format_time_as_string(timestamp: np.datetime64) -> str:
+    """
+    Converts a datetime64 object to a formatted string suitable for file naming.
+
+    Args:
+        timestamp (np.datetime64): The timestamp to format.
+
+    Returns:
+        str: A formatted string in 'YYYY-MM-DDTHHMMSS.mmm' format, 
+             compatible with Windows file naming.
+    """
+    formatted_time = str(timestamp.astype("datetime64[ms]"))[:21]
+    return formatted_time.replace(":", "")  # Remove colons for Windows compatibility
+
+def generate_file_name(start_time: np.datetime64, end_time: np.datetime64) -> str:
+    """
+    Generates a standardized file name for low-frequency DAS data.
+
+    Args:
+        start_time (np.datetime64): The start time of the data range.
+        end_time (np.datetime64): The end time of the data range.
+
+    Returns:
+        str: A file name in the format 'LFDAS_<start_time>_<end_time>.h5'.
+    """
+    start_time_str = _format_time_as_string(start_time)
+    end_time_str = _format_time_as_string(end_time)
+    return f"LFDAS_{start_time_str}_{end_time_str}.h5"
```

Comment:
Take a look at `dascore.utils.patch.get_default_patch_name` and see if it can replace this.

Reply:
I believe it can.